### PR TITLE
changes made following yesterday's review with JL: improved pre-selec…

### PR DESCRIPTION
**Pull request by skwong25**
1. Improved preselected dropdown options (no. of search results, sort criteria), 
2. Renamed components and elements for better specificity
3. Formatted search results for consistency incl. hyperlinks
4. Personalised browser tab title and favicon
5. Added static banner with imagery, icon and title
6. Filled space next to the Sort Criteria dropdown list by adding a check to verify no. of results returned is as per user selection ( sometimes it is less). 

Aware these changes have broken some tests, to be fixed tomorrow. 
README and Loading message to be looked at tomorrow also.

### CODE REVIEW

File: src/InputFieldComponent.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const InputField = (props) => {
 const numbers = [1,2,3,4,5,6,7,8,9,10]
 const instructions = [
```
Comment (fractalwrench) [inline src/InputFieldComponent.js:33]: may be worth making these bold for emphasis (not worth spending lots of time on)

File: src/MovieAppParentComponent.js [hunk 2]
```diff
@@ -21,41 +21,34 @@
...
-class App extends React.Component {
+class MovieApp extends React.Component {
 
     constructor(props) {
         super(props);
         this.state = { 
             keyword: "", 
             movie: null,
-            results: 1,
+            results: 10,
```
[the remark this answers is not in the document]
Comment (skwong25) [reply src/MovieAppParentComponent.js:51]: Believe that I can't do this within the Class Component so I've created the constant variable as an imported module - is this correct or more complicated that needs be?

File: src/MovieAppParentComponent.js [hunk 3]
```diff
@@ -111,28 +104,35 @@ class App extends React.Component {
...
+                        }}
+                    >
+                        <Button>
+                        <a href="https://www.bbcgoodfood.com/user/4010681/recipe/perfect-popcorn" target="_blank" rel="noopener noreferrer">
+                            <img alt="logo" src="../images/popcornFavicon.ico" />
+                        </a>
+                        </Button>
+                        <Typography variant="h6" className={classes.title}>Welcome to Movie App
+                        </Typography>
+                    </Toolbar>
```
[the remark this answers is not in the document]
Comment (skwong25) [reply src/MovieAppParentComponent.js:129]: Could not get this working. The guide there is for CSS, so not sure if that's why??  Googling Material UI mask image doesn't bring anything up. As a qucikfix, I placed the text on a black background to improve readability.

File: src/MovieAppParentComponent.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (fractalwrench) [reply src/MovieAppParentComponent.js]: in general it's probably overkill to create a module just for a constant, but fine to leave this either way for now

File: README.md [hunk 4]
```diff
@@ -1,68 +1,252 @@
...
+            category: "imdbID",
+            comparator(a,b) {
+                return sortMovies(a, b, "imdbID");
+            }
+        },
+```
+
+
+
+## Getting Started
```
Comment (fractalwrench) [inline README.md:187]: This should probably be the second section in the README (nearly at the top of the page)

File: README.md [hunk 5]
```diff
@@ -1,68 +1,252 @@
...
+## Motivation
 
-### `npm run eject`
+This is my first app I have created to put knowledge learnt of **Javascript** and **HTML** into practice. 
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+It allowed the opportunity to engage with **React**, using `<Components>` and **CSS** Styling to produce a user interface. 
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+It also involved establishing a **gitHub** workflow, using *pull requests*, *branches* and *documentation* to maintain a repository in collaboration with ~~my boyfriend~~ reviewers and contributors.
```
[the remark this answers is not in the document]
Comment (skwong25) [reply README.md:29]: wah. i was just fabricating an opportunity to use ~~strikethrough~~

File: README.md [hunk 6]
```diff
@@ -1,68 +1,252 @@
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+# RB's Movie App #
```
[the remark this answers is not in the document]
Comment (skwong25) [reply README.md:3]: I will tell them it stands for 'Really Businesslike'

[the remark this answers is not in the document]
Comment (skwong25) [reply README.md:3]: or 'Results-driven and Busy'

**Pull request merged**